kivy/uix/widget.py

```diff
@@ -725,7 +725,7 @@ def export_to_png(self, filename, *args, **kwargs):
 
                 .. versionadded:: 1.11.0
         '''
-        self.export_as_image(*args, **kwargs).save(filename, flipped=False)
+        self.export_as_image(*args, **kwargs).save(filename, fmt=kwargs.pop('fmt','png'), flipped=False)
```

Comment:
I think i would rather put fmt='png' in the method signature, and document that usage in the docstring, especially as export_as_image doesn't need it, so no need to pass it.

I'm not sure why we do it this way for export_as_image with scale, seems like a bad pattern to me.

Reply:
Was trying to save the need for documenting, but agree though that this looks out of place.
Would you default to png or None and require the fmt to be stated for BytesIO.
I'll edit - may need to double check valid fmt formats for documenting

Reply:
I would go for the one that doesn't break previous usage, which i think actually would be `fmt=None`? as currently if you pass a filename but not fmt, the format is going to be deduced from the filename, while if we force fmt='png' by default, then it's going to save as png, even if the filename is `.jpeg`, which would require any such usage to now pass the desired value of `fmt`.

Reply:
I sort of took the method name literally there, expecting to get a png format back, will change.